test/runtime/ferguson/testatomics.test.c

```diff
@@ -74,6 +74,8 @@ int main(int argc, char** argv)
 {
   atomic_bool flag;
 
+  memset(&flag, 0, sizeof(flag));
```

Comment:
You need to make the test "better" to quiet it for valgrind, and if atomic_init_bool quiets valgrind, I can't imagine why you wouldn't want to use the runtime provided "initializer" over rolling your own with a memset.